readme.txt

```diff
@@ -33,10 +33,10 @@ From there:
 
 == Changelog ==
 
-= 4.17.1 =
+= 4.18 =
 
-#### 👾 Bug Fixes
-- Fix issue Mobile Banking and Rabbit LINE Pay not showing.
```

Comment:
maybe we shouldn't remove the changelog of an older release, rather we should add to it

Reply:
didn't see this got replaced as well
 just create branch base on v4.17 so after merge 4.17.1 is got replace by 4.18
nice catch 👍  
Fixed -> https://github.com/omise/omise-woocommerce/pull/254/commits/778ea0b1b6a14c945f3d6b21011ee009060c713e